src/lib/openapi/spec/project-stats-schema.ts

```diff
@@ -18,30 +18,57 @@ export const projectStatsSchema = {
     properties: {
         avgTimeToProdCurrentWindow: {
             type: 'number',
+            example: 10,
+            description:
+                'Average time to production for features in the current window (last 30 days)',
```

Comment:
Is this a correct understanding of it? 

```suggestion
                'The average time from when a feature was created to when it was enabled in the "production" environment during the current window",
```